src/sgl/device/command.cpp

```
@@ -295,13 +295,26 @@ void RayTracingPassEncoder::end()
+    Slang::ComPtr<rhi::ICommandEncoder> rhi_command_encoder
+)
     : DeviceChild(std::move(device))
+    , m_queue(queue)
+    , m_recording_id(recording_id)
     , m_rhi_command_encoder(std::move(rhi_command_encoder))
     , m_open(true)
 {
 }
 
+CommandEncoder::~CommandEncoder()
+{
```

Cppcheck (2.20.0)

[style] The function 'supported_shader_model' is never used.

(unusedFunction)